src/schedule.ts

```diff
@@ -44,10 +56,10 @@ const flush = (): void => {
 }
 
 export const shouldYield = (): boolean => {
-  if (options.sync) return false
-  return (
+  const isInputPending =
     (navigator as any)?.scheduling?.isInputPending() || getTime() >= deadline
```

Comment:
I've run some benchmarks and it seems that chrome is actually doing worse with `isInputPending`:

Firefox create 1k rows (20x): ~573ms
Chrome (without isInputPending) create 1k rows (20x) : ~642ms
Chrome (with isInputPending) create 1k rows (20x) : ~730ms

It would be nice if someone could confirm this as well;
Another case with it is that `isInputPending` function call creates ugly code that all other browsers needs to handle:
```
"use strict";
var _a, _b;
const isInputPending = ((_b = (_a = navigator) === null || _a === void 0 ? void 0 : _a.scheduling) === null || _b === void 0 ? void 0 : _b.isInputPending()) || getTime() >= deadline;
```

Reply:
In fact, there is no need to isInputPending, and the calculation using time slice is enough. You can delete it.
https://github.com/WICG/scheduling-apis 
This is a new set of scheduling APIs, but it is not yet mature. 
We can remove them first and then add it when the time is ripe.